WooCommerce/Classes/System/SessionManager.swift

```diff
@@ -95,6 +95,17 @@ final class SessionManager: SessionManagerProtocol {
         }
     }
 
+    /// Roles for the default Store Site.
+    ///
+    var defaultRoles: [String] {
```

Comment:
Maybe we can have this be `[User.Role]`? I imagine if we want to use it in more places, the `enum` will almost always be more useful than the raw strings. It doesn't seem like a big deal right now, so I'm OK with it either way.

Reply:
That was my initial approach, but sadly, `SessionManager` stores its information using `UserDefaults` so we are limited to use scalar? types.

Although, now that I think about it I could do that conversion in the session manager 🤔

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/4735/commits/b2d0622102a4701a1d8e4c965bec6ca7a68aa1e1 

Thanks for making me double-check!